src/content/manifest/hybrid-development.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Hybrid development
+description: Hybrid development allows you to develop and run a service locally while the other components of your app run in the cloud.
+sidebar_label: Hybrid development
+id: hybrid-development
+---
+
+import Image from '@theme/Image';
+
+Okteto offers two different approaches to service development: `sync` or `hybrid` mode. This mode is configured in your Okteto manifest and affects the behavior of [`okteto up`](/reference/cli/#up).
```

Comment:
@omnomagonz not in this PR, but I think this page should explain how dev containers work. And as part of it, explain how `sync` and `hybrid` modes work